go/ql/test/library-tests/semmle/go/dataflow/ExternalFlowInheritance/test.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
```

Comment:
Recommend using the SPURIOUS attribute throughout (and if applicable MISSING, though I don't think there any any missing results) to identify what ought to be fixed